.github/workflows/run_benchmarks_over_history.yml

```
@@ -27,8 +27,8 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: 3.8
-      - name: Install tox and asv
-        run: pip install -U pip "tox<4" asv
+      - name: Install nox and asv
+        run: pip install -U pip "nox" asv
```

```suggestion
      - name: Install nox and asv
        run: pip install -U pip nox asv
```

We can remove the double quotes since we don't need to install a specific version of `nox`